README.md

```diff
@@ -166,6 +166,8 @@ Optional settings:
     - `NONE`: No text added. (Default.)
     - `CONFIGURATION`: The name of the configuration which the dependency belongs to (e.g. "
       implementation", "compileOnly", "jsMain").
+- **excludeConfigurationNames**:
```

Comment:
Please add an example of usage in the code blocks above.  Like [here](https://github.com/iurysza/module-graph/blob/5f3117bc52144ba1542155eab3805a5b9d5ee729/README.md?plain=1#L68)

Reply:
added https://github.com/iurysza/module-graph/pull/25/commits/2587876249e8e69e50a4ae7b586605d790a79ae9